documentation/modules/security/con-certificate-authorities.adoc

```diff
@@ -26,3 +26,15 @@ You can configure the validity period of:
 
 * Cluster CA certificates in `Kafka.spec.clusterCa.validityDays`
 * Client CA certificates in `Kafka.spec.clientsCa.validityDays`
+
+== Options for installing your own CA certificates
```

Comment:
I think this mixes two things which are separate concepts:
* Suplying your own CA certificates
* Suplying server certficates for the TLS and External listeners.
They should either no be mixed into the same chapter named after nly one of these or we need to find new name.

Reply:
Thanks for the clarification. Tom thought that we needed a short concept to distinguish between these two separate but related tasks.

I'll re-word and ask you to re-review.